lib/src/components/NxSubmitMask/NxSubmitMask.scss

```diff
@@ -6,11 +6,12 @@
  */
 @import '../../scss-shared/_nx-variables';
 @import '../../scss-shared/_nx-container-helpers';
+@import '../../scss-shared/_nx-text-helpers';
 
 .nx-submit-mask {
   align-items: center;
 
-  background-color: rgba($nx-off-white, 0.9);
+  background-color: rgba(#555, 0.75);
```

Comment:
I don't think this is technically valid css syntax, but scss seems to be converting it so I guess it's fine

Reply:
> I don't think this is technically valid css syntax, but scss seems to be converting it so I guess it's fine

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/12f0e8dbc13f4e55843306be3219208c4a9eaba3

Reply:
> > I don't think this is technically valid css syntax, but scss seems to be converting it so I guess it's fine
> 
> Fixed here: [12f0e8d](https://github.com/sonatype/sonatype-react-shared-components/commit/12f0e8dbc13f4e55843306be3219208c4a9eaba3)

This is a better fix, more consistent with how we do it elsewhere: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/eebce7f8a2e4002495a1b1d2ece4669d033a5c15